docs/prompting/zero_shot/emotion_prompting.md

```diff
@@ -1,7 +1,46 @@
 ---
-title: ""
+title: "Emotion Prompting"
 description: ""
-keywords: ""
+keywords: "prompt, prompting, prompt engineering, llm, gpt, openai, emotion prompting, model, AI, python, instructor, zero-shot"
 ---
 
-[wip]
+Emotion prompting<sup><a href="https://arxiv.org/abs/2307.11760">1</a></sup> uses phrases that have psychological relevance to humans in the LLM prompt. This may lead to improved performance on benchmarks and open-ended text generation.<sup><a href="https://arxiv.org/abs/2406.06608">\*</a></sup>
+
+```python
+import openai
```

Comment:
Simplify the example and use admonitions for the emotional prompting portion

see : https://squidfunk.github.io/mkdocs-material/reference/admonitions/